Formula/rdkit.rb

```diff
@@ -5,6 +5,7 @@ class Rdkit < Formula
   sha256 "85cec9618e7ef6365b9b908ed674c073d898b6627521cc7fd8c2e05fea8a5def"
   license "BSD-3-Clause"
   head "https://github.com/rdkit/rdkit.git"
+  revision 1
```

Comment:
I don't think the revision bump is needed.

Reply:
`numpy` was correctly found in the previous installation on macOS.
But an additional option `-DPYTHON_NUMPY_INCLUDE_PATH=#{numpy_include}` is required for installation on Linux.